.gitignore

```diff
@@ -6,3 +6,4 @@ coverage
 release-out
 .certs
 .tmp
+.vscode
```

Comment:
Ah, meant to comment on this one (https://github.com/moby/buildkit/commit/22c87dd8f53e2359f5e7faaf4f7c2453612c1f3e), but I guess the same applies to .dockerignore 🤔